/tmp/glance/doc/source/conf.py:188:25: DUO106 insecure use of "os" module
/tmp/glance/glance/common/crypt.py:25:1: DUO133 use of "Crypto" module is insecure
/tmp/glance/glance/common/crypt.py:26:1: DUO133 use of "Crypto" module is insecure
/tmp/glance/glance/common/crypt.py:27:1: DUO133 use of "Crypto" module is insecure
/tmp/glance/glance/common/crypt.py:44:14: DUO102 insecure use of "random" module, prefer "random.SystemRandom"
/tmp/glance/glance/openstack/common/jsonutils.py:40:1: DUO107 insecure use of XML modules, prefer "defusedxml"
/tmp/glance/glance/store/filesystem.py:228:20: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/test_bin_glance_cache_manage.py:62:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/test_cache_middleware.py:70:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/test_cache_middleware.py:253:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_copy_to_file.py:125:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_copy_to_file.py:126:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_copy_to_file.py:147:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_copy_to_file.py:148:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_copy_to_file.py:235:30: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_copy_to_file.py:236:30: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_misc.py:64:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_ssl.py:160:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_ssl.py:212:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_ssl.py:213:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_ssl.py:429:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/functional/v1/test_ssl.py:452:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/test_filesystem_store.py:89:29: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/test_image_cache.py:379:15: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/test_s3_store.py:74:24: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/test_s3_store.py:234:29: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/test_s3_store.py:280:33: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/v1/test_api.py:2888:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/v1/test_api.py:2912:26: DUO130 insecure use of "hashlib" module
/tmp/glance/glance/tests/unit/v1/test_api.py:2944:24: DUO130 insecure use of "hashlib" module
